3-networks-svpc/modules/shared_vpc/firewall.tf

```diff
@@ -61,42 +61,6 @@ module "firewall_rules" {
           ]
         }
       }
-    ],
-    !var.enable_all_vpc_internal_traffic ? [] : [
-      {
-        priority       = "10000"
-        direction      = "EGRESS"
-        action         = "allow"
-        rule_name      = "fw-${var.environment_code}-shared-base-10000-e-a-all-all-all"
-        description    = "Allow all egress to the provided IP range."
-        enable_logging = var.firewall_enable_logging
-        match = {
-          dest_ip_ranges = module.main.subnets_ips
-          layer4_configs = [
-            {
-              ip_protocol = "all"
-            },
-          ]
-        }
-      }
-    ],
-    !var.enable_all_vpc_internal_traffic ? [] : [
-      {
-        priority       = "10001"
-        direction      = "INGRESS"
-        action         = "allow"
-        rule_name      = "fw-${var.environment_code}-shared-base-10001-i-a-all"
-        description    = "Allow all ingress to the provided IP range."
-        enable_logging = var.firewall_enable_logging
-        match = {
-          src_ip_ranges = module.main.subnets_ips
-          layer4_configs = [
-            {
-              ip_protocol = "all"
-            },
-          ]
-        }
-      }
```

Comment:
why this rules where removed?

Reply:
my mistake, I related it wrongly to the shared-base project, not to internal network traffic, I will add it again.